libs/spark/README.md

```diff
@@ -0,0 +1,66 @@
+# spark
+
+This library was generated with [Nx](https://nx.dev).
+
+## Running unit tests
+
+Run `nx test spark` to execute the unit tests via [Jest](https://jestjs.io).
+
+## Prenda Spark
+
+- [Documentation](#documentation)
+- [Development](#development)
+- [Testing](#testing)
+- [WIP and TODOS](#wip-and-todos)
+- [Install](#install)
+- [Usage](#usage)
+
+---
+
+## Documentation
+
+> Official Docs: https://prenda-school.github.io/prenda-documentation/
+
+Prenda Spark extends Material UI react components (https://material-ui.com/) by overriding select styles to match the Prenda Design System philosophy
+
+## Development
+
+1. Clone the repository.
+2. Ensure you have Node Version Manager installed on your system ([see instructions](https://github.com/nvm-sh/nvm#installing-and-updating)).
+3. Run `> nvm use` to ensure specified node version is used.
+4. Run `> npm ci` to install dependencies.
+5. Run `> nx run spark:storybook` to trigger nx to run the spark project in develop mode (open `http://localhost:4200/` in your default browser to view.)
```

Comment:
This url should be `http://localhost:4400/`

Reply:
I don't why my brain keeps telling me it is 4200 🙃